docs/wiki/The CI environment - Static validation.md

```diff
@@ -54,6 +54,10 @@ In particular, each resource's API version is compared with those currently avai
 
 This test also leverages the [`utilities/pipelines/staticValidation/module.tests.ps1`](https://github.com/Azure/ResourceModules/blob/main/utilities/pipelines/staticValidation/module.tests.ps1) script.
 
+To test the API versions, the test leverages the file `utilities/src/apiSpecsList.json` file as a reference for all API versions available for any used Provider Namespace & Resource Type.
+
+> **NOTE:** This functionality has a dependency on the [AzureAPICrawler](https://www.powershellgallery.com/packages/AzureAPICrawler/0.1.2) PowerShell module.
```

Comment:
```suggestion
> **NOTE:** This functionality has a dependency on the [AzureAPICrawler](https://www.powershellgallery.com/packages/AzureAPICrawler) PowerShell module.
```
Otherwise we risk to get stale (this PR itself uses 0.2.0 instead)